themes/pure/pure.theme.bash

```diff
@@ -31,11 +31,18 @@ pure_prompt() {
     ps_root_mark="${red} # ${normal}"
     ps_path="${yellow}\w${normal}";
 
+	# Virtualenv
+	if [[ $VIRTUAL_ENV != "" ]]; then
+		venv="(${VIRTUAL_ENV##*/})"
+	else
```

Comment:
You could avoid the `else` block if you initialized the `venv` variable before the `if`:

```bash
local venv=""
```